Dockerfile

```diff
@@ -0,0 +1,19 @@
+# Steps to Use
+# 1. Download Docker
+# 2. Build with `docker build -t {image tag} .`
+# 3. Run `docker run -t -i -v {local absolute directory}:{container directory} {image tag name} /bin/bash`
+
+FROM ubuntu
+
+RUN apt-get update
+RUN apt-get -y upgrade
```

Comment:
Hi @kgryte, thanks for clarifying that most systems already have `pandoc`, I was using your script as part of a project to get more comfortable with Docker so I agree it's overkill for most people. Thanks for the feedback on the Dockerfile above.